policy/common/methods.go

```diff
@@ -34,10 +34,28 @@ type Methods struct {
 	GithubReviewState pull.ReviewState `yaml:"-" json:"-"`
 }
 
+type CandidateType string
+
+const (
+	ReviewCandidate  CandidateType = "review"
+	CommentCandidate CandidateType = "comment"
+)
+
+type DiscardReason string
+
+const (
+	EditedCandidate            DiscardReason = "edited"
+	InvalidatedByPushCandidate DiscardReason = "invalidated by pushing another commit"
+)
+
 type Candidate struct {
-	User         string
-	CreatedAt    time.Time
-	LastEditedAt time.Time
+	Type             CandidateType
+	ID               string
```

Comment:
nit: I'd call this `ReviewID` to be clear it is not set for other candidate types